dask/bytes/pyarrow.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import print_function, division, absolute_import
+
+import re
+import fnmatch
+import posixpath
+
+from ..base import tokenize
+
+import pyarrow as pa
+
+
+class HDFS3Wrapper(pa.filesystem.DaskFileSystem):
```

Comment:
This is for arrow's libhdfs3?
I know this is just moved from above, but it's not obvious to me what it's doing, while the class below seem to have all the required behaviour.

Reply:
This is for wrapping `hdfs3`'s filesystem to be used inside pyarrow. I added a comment that should clarify this.